packages/components/tests/integration/components/hds/tabs/index-test.js

```diff
@@ -7,7 +7,14 @@ module('Integration | Component | hds/tabs/index', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders the component', async function (assert) {
-    await render(hbs`<Hds::Tabs />`);
+    await render(hbs`
+      <Hds::Tabs id="test-tabs" as |T|>
+        <T.Tab data-test="tab">One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel data-test="panel">Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+  `);
```

Comment:
The test was failing before I added the child elements

Reply:
OK

(in this case, you can remove the `data-test` attributes, are not used)